### PR TITLE
Add messages used to test batching

### PR DESCRIPTION
**Pull request by benpigchu**

### CODE REVIEW

**Comment by ice1000** [comment]
Could you please fix the conflicts?

**Comment by benpigchu** [comment]
@ice1000 not now. Keep this open at this moment.

File: proto/tikvpb.proto [hunk 1]
```diff
@@ -125,15 +127,29 @@ message BatchCommandsResponse {
...
+message BatchCommandTestRequest {
+    // Id of the test request
+    uint64 test_id = 1;
+    // The time between receiving the request and sending the response
+    uint64 delay_time = 2;
+}
+
+message BatchCommandTestResponse {
+    // Id of the test response, should be same with the one in the request
+    uint64 test_id = 1;
```
Comment (hicqu) [inline proto/tikvpb.proto]: The response is generated out of gRPC threads. So you can add
* `reply_time` which is for when the response is generated
* `reply_send_time` which is for when the response is sent out

**Comment by hicqu** [comment]
rest LGTM. After we get 2 LGTM, I think we can push a `3.0` branch to pingcap/kvproto.

**Comment by ice1000** [comment]
image
FYI You can actually change the base branch.

**Comment by hicqu** [comment]
LGTM. Now we can consider how to use it.

**Pull request closed without merging**